components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,291 +10,788 @@ toc = true
-
-### Getting a Policy
+```bash
+curl -sH "api-token: $TOKEN" -d '<your JSON here>' ...
+```
 
-In order to fetch a policy on the command-line, you must know its ID.
-This may be obtained by first fetching the list of all policies; see [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}).
-Find the `id` field for the policy of interest--for example, the ID from the default administrator policy is `administrator-access`.
-Plug that in to the URL as shown:
+If the payload is larger, it is often convenient to store it in a file, then pass that file, e.g.,
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/administrator-access?pretty
+curl -sH "api-token: $TOKEN" -d @policy.json ...
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab, and then select an individual policy to open its details.
+## Projects Property
 
-### Creating a Policy
+Teams, tokens, roles, and policies all contain a top-level `projects` property; you will see that in the example JSON for each resource in the following sections.
+This property associates the particular IAM resource with one or more projects.
+Users must have permissions for those projects to be able to view or modify those resources.
+If the `projects` property is left empty, that resource is categorized as *unassigned*.
+
+To create those permissions for users, there is a separate and distinct `projects` property on each policy statement.
+This `projects` property may **not** be empty; you must either specify specific projects or use a wildcard (`*`) to indicate all projects.
+
+Type               | May be empty ?    | Allows "*" ?         | May specify (unassigned) ?
+-------------------|-------------------|----------------------|---------------------------
+top-level projects | empty allowed     | wildcard not allowed | (unassigned) not allowed
+statement projects | empty not allowed | wildcard allowed     | (unassigned) allowed
+
+## Policies
+
+HTTP request              | Description
+--------------------------|------------
+GET /policies             | lists both *Chef-managed* and *Custom* policies
+GET /policies/{**id**}    | gets the specified policy
+POST /policies            | creates a *Custom* policy
+PUT /policies/{**id**}    | updates a *Custom* policy
+DELETE /policies/{**id**} | deletes a *Custom* policy
 
-Create a policy by composing JSON with all the necessary policy data.
+### Example Policy
 
-This example policy is made up of two statements, each enforcing a different set of permissions.
-The first policy statement grants users list view access.
-The second statement grants editor access via the `editor` role.
-You can see the actions contained in the `editor` role in the [Roles section]({{< relref "iam-v2-api-reference.md#roles" >}}).
+This policy shows two statements, each enforcing a different set of permissions.
```

Comment:
[nit] I bet this has already been discussed but `This policy shows two statements` is weird to me. Maybe `This policy contains two statements`?